### PR TITLE
[ci] add reef team as code ownership of ray_ci

### PR DESCRIPTION
**Pull request by can-anyscale**
Add reef team as code ownership of ci/ray_ci

Test:

### CODE REVIEW

**Comment by anyscalesam** [comment]
@can-anyscale can we close this?

**Comment by can-anyscale** [comment]
@anyscalesam you don't like this one? it's still under review

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -44,6 +44,7 @@
 # Docker image and build scripts.
 /ci/build/build-docker-images.py @ray-project/ray-core @aslonnie @can-anyscale
 /ci/docker @aslonnie @can-anyscale
+/ci/ray_ci @aslonnie @can-anyscale @khluu
```
Comment (aslonnie) [inline .github/CODEOWNERS:47]: maybe change to @ray-project/ray-ci-reviewers or @ray-project/ray-ci ?

**Comment by anyscalesam** [comment]
Ah sorry - I misread and thought this was super old. You good (carry on 🙃 )

**Comment by can-anyscale** [comment]
change the owner to @ray-project/ray-ci

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -44,6 +44,7 @@
 # Docker image and build scripts.
 /ci/build/build-docker-images.py @ray-project/ray-core @aslonnie @can-anyscale
 /ci/docker @aslonnie @can-anyscale
+/ci/ray_ci @ray-project/ray-ci
```
Comment (aslonnie) [inline .github/CODEOWNERS:47]: hmm... sorry, but maybe the `/ci/docker` one too?

Comment (aslonnie) [reply .github/CODEOWNERS:47]: and the part in `/ci/build/build-docker-images.py`?

**Review by aslonnie — APPROVED** [review]

**Comment by can-anyscale** [comment]
@aslonnie's comments

**Pull request merged**